local-dev/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  dynamo:
+    container_name: local-dynamodb
+    image: amazon/dynamodb-local
+    networks:
+      - local-dynamodb
+    ports:
+      - "8000:8000"
+    volumes:
+      - dynamodata:/home/dynamodblocal
+    working_dir: /home/dynamodblocal
+    command: "-jar DynamoDBLocal.jar -sharedDb -dbPath ."
+
+networks:
+  local-dynamodb:
+    name: local-dynamodb
+
+volumes:
+  dynamodata: {}
```

Comment:
Is it worth mounting the shared DB under `~/.gu/janus-app` on the host? I don't feel strongly, but would mean that you don't have to recreate the db tables any time you update the container.

Reply:
Yes that's a good idea

Reply:
Done in 5ed9d687a0c1feef1877e22188b4e7db6d1b7585